kieker-analysis/build.gradle

```diff
@@ -10,12 +10,16 @@ targetCompatibility = 1.8
 dependencies {
 	api project(':kieker-common')
 	api project(':kieker-monitoring')
+	api project(':kieker-model')
 
-	implementation resolveLibrary('javax.jms-*.jar')				// old library; not available via maven repo
 	api 'org.apache.cxf:cxf:2.6.17'			// cxf-bundle
 	api resolveLibrary('org.eclipse.emf.common_*.jar')
 	api resolveLibrary('org.eclipse.emf.ecore.xmi_*.jar')
 	api resolveLibrary('org.eclipse.emf.ecore_*.jar')
+	
+	implementation group: 'javax.jms', name: 'javax.jms-api', version: '2.0.1'
+    implementation group: 'javax.servlet', name: 'javax.servlet-api', version: '4.0.1'
```

Comment:
Not sure whether this is wrongly displayed in GitHub or whether the indentation is wrong here.

Reply:
I'll update the package-info.java files and check the formatting.